components/landing/VersionInfo.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <section class="qiskit-version-info">
+    <a
```

Comment:
Just a comment: When we will merge this branch with the master one, we should remember to use the AppLink component instead of `<a>`